package.json

```diff
@@ -92,6 +92,7 @@
     "hapi-swagger": "github:brave/hapi-swagger",
     "http-request-signature": "0.0.2",
     "inert": "^4.2.1",
+    "isuuid": "^0.1.0",
```

Comment:
I rarely say this, but I think it would be better if you just copy-pasted the 2 lines from the repository instead of adding it as a dependency: https://github.com/nescalante/isuuid/blob/master/index.js#L6-L8

Rationale:
* It's only 2 lines, can be combined into 1 line easily.
* The package hasn't been updated since 2015
* Adding a package dependency adds some risk that it'll be compromised/broken, like what happened with leftpad. https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/

Reply:
happy to!